processing/src/main/java/io/druid/segment/data/CompressedObjectStrategy.java

```diff
@@ -85,6 +92,22 @@ public Compressor getCompressor()
       {
         return UncompressedCompressor.defaultCompressor;
       }
+    },
+    /*
+    This value indicate no compression strategy should be used, and compression should not be block based
+     */
+    NONE((byte) 0xFE) {
+      @Override
+      public Decompressor getDecompressor()
+      {
+        return UncompressedDecompressor.defaultDecompressor;
```

Comment:
Changed this to throw exception. Also modified some tests for compressed float and ints to not use NONE since they don't support properly reading NONE compression type. It makes sense since we never want to use uncompressed format for NONE as it require a new Compression strategy to be created in the future if we want float or int to support NONE.